CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ Package-specific changes not released in any SDK will be added here just before
 - Updated `@react-native-community/netinfo` from `6.0.0` to `6.0.2`. ([#14329](https://github.com/expo/expo/pull/14329) by [@cruzach](https://github.com/cruzach))
 - Updated `react-native-webview` from `11.6.2` to `11.13.0`. ([#14301](https://github.com/expo/expo/pull/14301) by [@kudo](https://github.com/kudo))
 - Updated `react-native-lottie` from `4.0.2` to `4.0.3`. ([#14331](https://github.com/expo/expo/pull/14331) by [@cruzach](https://github.com/cruzach))
+- Updated `@react-native-community/viewpager` from `5.0.11` to `react-native-pager-view@5.4.1`. ([#14348](https://github.com/expo/expo/pull/14348) by [@cruzach](https://github.com/cruzach))
```

Comment:
```suggestion
- Updated `@react-native-community/viewpager` from `5.0.11` to `5.4.1`. ([#14348](https://github.com/expo/expo/pull/14348) by [@cruzach](https://github.com/cruzach))
```
❓ 😄

Reply:
well we also moved from `@react-native-community/viewpager` to `react-native-pager-view`, so thought I'd call that out here 😅